client/src/components/BuySell.jsx

```diff
@@ -0,0 +1,1341 @@
+text-decoration-line: none;
+text-decoration-style: solid;
+text-size-adjust: 100%;
+white-space: nowrap;
+width: 78.1719px;
+-webkit-box-align: center;
+-webkit-box-direction: normal;
+-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+`;
+
+const MarketPriceSpan = styled.span`
+box-sizing: border-box;
+color: rgb(255, 255, 255);
+display: block;
+font-family: "DIN Pro", -apple-system, system-ui, sans-serif;
+font-size: 13px;
+font-weight: 500;
+height: 19px;
+letter-spacing: 0.25px;
+line-height: 19px;
+overflow-x: hidden;
+overflow-y: hidden;
+text-align: right;
+text-overflow: ellipsis;
+text-size-adjust: 100%;
+white-space: nowrap;
+width: 47.9219px;
+-webkit-box-direction: normal;
+-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+`;
+
+const EstimatedCostRow = styled.div`
+border-bottom-color: rgb(14, 13, 13);
+border-left-color: rgb(14, 13, 13);
+border-right-color: rgb(14, 13, 13);
+border-top-color: rgb(14, 13, 13);
+border-top-style: solid;
+border-top-width: 1px;
+box-sizing: border-box;
+color: rgb(255, 255, 255);
+display: block;
+font-family: "DIN Pro", -apple-system, system-ui, sans-serif;
+font-size: 13px;
+font-weight: 500;
+height: 49px;
+letter-spacing: 0.25px;
+line-height: 19px;
+padding-right: 12px;
+text-align: left;
+text-size-adjust: 100%;
+width: 227.766px;
+-webkit-box-direction: normal;
+-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+`;
+
+const EstimatedCostLabel = styled.label`
+align-items: center;
+box-sizing: border-box;
+color: rgb(255, 255, 255);
+cursor: default;
+display: flex;
+font-family: "DIN Pro", -apple-system, system-ui, sans-serif;
+font-size: 13px;
+font-weight: 500;
+height: 48px;
+justify-content: space-between;
+letter-spacing: 0.25px;
+line-height: 19px;
+min-height: 48px;
+text-align: left;
+text-size-adjust: 100%;
+white-space: nowrap;
+width: 215.766px;
+-webkit-box-align: center;
+-webkit-box-direction: normal;
+-webkit-box-pack: justify;
+-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+`;
+
+const ECDiv1 = styled.div`
+box-sizing: border-box;
+color: rgb(255, 255, 255);
+cursor: default;
+display: block;
+font-family: "DIN Pro", -apple-system, system-ui, sans-serif;
+font-size: 13px;
+font-weight: 500;
+height: 19px;
+letter-spacing: 0.25px;
+line-height: 19px;
+margin-right: 12px;
+min-width: 48px;
+overflow-x: hidden;
+overflow-y: hidden;
+text-align: left;
+text-overflow: ellipsis;
+text-size-adjust: 100%;
+white-space: nowrap;
+width: 110px;
+-webkit-box-direction: normal;
+-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+`;
+
+const ECDiv2 = styled.div`
+box-sizing: border-box;
+color: rgb(255, 255, 255);
+cursor: default;
+display: block;
+font-family: "DIN Pro", -apple-system, system-ui, sans-serif;
+font-size: 13px;
+font-weight: 500;
+height: 19px;
+letter-spacing: 0.25px;
+line-height: 19px;
+text-align: left;
+text-size-adjust: 100%;
+white-space: nowrap;
+width: 45px;
+-webkit-box-direction: normal;
+-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+`;
```

Comment:
where is all of this CSS coming from? did you write it all? it seems a little crazy that we have 1100 lines of CSS in this JS file